pl_bolts/models/vision/unet.py

```diff
@@ -17,21 +17,23 @@ class UNet(nn.Module):
 
     Args:
         num_classes: Number of output classes required
+        input_channels: Number of channels in input images (default 3)
         num_layers: Number of layers in each side of U-net (default 5)
         features_start: Number of features in first layer (default 64)
         bilinear (bool): Whether to use bilinear interpolation or transposed convolutions (default) for upsampling.
```

Comment:
```suggestion
        bilinear: Whether to use bilinear interpolation or transposed convolutions (default) for upsampling.
```
mind removing ` (bool)` for consistency?

Reply:
good idea - thanks for pointing it out! :)